### PR TITLE
updated readme.md and contributers.md

### PR DESCRIPTION
**Pull request by priyanshu9899**
### Related Issue
issue #8 -> updated the images link and added setup instruction in readme.md and contributers.md 

### Proposed Changes 
- updated images link in readme.md
- added setup instruction in contributing.md

### Checklist 
- [x] Documentation of the code 
- [x] Add your name to the CONTRIBUTORS file 
- [x] Testing your code in the local machine

### Screenshots
Screenshot 2023-10-21 at 2 45 09 PM
Screenshot 2023-10-21 at 2 45 38 PM

### CODE REVIEW

File: CONTRIBUTING.md
Comment (paardarshee) [inline CONTRIBUTING.md:1]: Edit this
with 
```sh
npm i
npm start
```

as someone can copy paste the codes directly

File: README.md
Comment (paardarshee) [inline README.md:1]: - >Here in line 19, 
  >Update it, as it is no longer just an HTML, CSS, and JS Website.
  >Now It is using React with Vite and Tailwind

- >In line 28,29 Add those scripts you added in Contributing.md
  > after git clone ...., Add this scripts:-
  >```sh 
  > cd GDSCAEC-WEBSITE
  > npm i
  > npm start
  > ```

- > After the git commit msg (line 32) add Sync your local clone in our repositories section on github and  Add this script `git pull origin your_branch_name` to resolve your merge conflict locally

**Review by paardarshee — CHANGES_REQUESTED** [review]
Update these Changes

**Comment by priyanshu9899** [comment]
pls add hacktoberfest accepted label in my pr

File: README.md
Comment (priyanshu9899) [reply README.md:1]: sure

**Review by paardarshee — APPROVED** [review]

**Review by paardarshee — APPROVED** [review]

**Pull request merged**